shared/actions/chat.js

```diff
@@ -111,6 +111,10 @@ const _messageOutboxIDSelector = (state: TypedState, conversationIDKey: Conversa
 const _pendingFailureSelector = (state: TypedState, outboxID: OutboxIDKey) => state.chat.get('pendingFailures').get(outboxID)
 const _devicenameSelector = (state: TypedState) => state.config && state.config.extendedConfig && state.config.extendedConfig.device && state.config.extendedConfig.device.name
 
+function _tmpFileName (isHdPreview: boolean, messageID: ?MessageID, filename: string) {
+  return `kbchat-${isHdPreview ? 'hdPreview' : 'preview'}-${messageID || ''}-${filename}`
```

Comment:
doesn't this need conversationid also? you could collide (rare but possible right)